packages/nx/schemas/nx-schema.json

```
@@ -745,7 +745,28 @@
           "manifestRootsToUpdate": {
             "type": "array",
             "items": {
-              "type": "string"
+              "oneOf": [
+                {
+                  "type": "string",
+                  "description": "Path to the directory containing manifest files to update. Supports placeholders like {projectRoot} and {projectName}."
+                },
+                {
+                  "type": "object",
+                  "properties": {
+                    "path": {
+                      "type": "string",
+                      "description": "Path to the directory containing manifest files to update. Supports placeholders like {projectRoot} and {projectName}."
+                    },
+                    "preserveLocalDependencyProtocols": {
+                      "type": "boolean",
+                      "description": "Whether to preserve local dependency references using protocols like 'workspace:' or 'file:'. Set to false for dist files that need to be published.",
```

Ambiguously written, could be taken to mean that it _is_ set to be that

```suggestion
                      "description": "Whether to preserve local dependency references using protocols like 'workspace:' or 'file:'. Set this to false for dist files that need to be published if not using a package manager that swaps references at publish time like pnpm or bun.",
```